### DISCUSSION TITLE
Improve consistency when using relative line numbers

### DISCUSSION THREAD

**Discussion by Jedsek** [helix-editor/helix#10355] [Ideas]
I'm new in vim and helix. After I added `line-number = "relative"` to my `config.toml`, I find that `goto` did not perform as I expected. And vim

Could we add negative count support for `:g <count>`, `:<count>`, `<count> G` ? 

For example: 

```
2 theme = "nord"
1
3 [editor]
1 line-number = "relative"
2 scrolloff = 0
```

First, helix should check whether you have enabled `relative line numbers`.

You could type `:g -2` to jump to `theme = "nord"` if you enabled `rnu`. 
(Or jump to `scrolloff = 0` by typing `:g 2`)

**Comment by CptPotato** [comment]
I'm not sure if this is the default or set in the configuration I'm using but this is the behavior for me in vim:
- `:2` jump to line 2
- `:-2` / `:+2` jump two lines up/down

While I like these commands, for relative navigation I find using these motions in helix more convinient: <kbd>2</kbd><kbd>j</kbd> / <kbd>2</kbd><kbd>k</kbd> (also works with the arrow keys)

**Comment by Jedsek** [comment]
@CptPotato  Yeah, I knew it before I opened this issue. Thank you.

**Comment by pickfire** [comment]
Is this closed then?

**Comment by Jedsek** [comment]
@pickfire  `<n> j/k` could't jump to the line start, but `:<n>` could. 
This leads to the fact that I have to type `<n> j/k ` and  `gh` for the same behaviour as `:<n>` in absolute line numbers

**Comment by pickfire** [comment]
Yes, I believe that should be the behavior. `:<n>` is supposed to go to the line and probably start as well. But `<n> j/k` should go up or down but the current column should be kept.

**Comment by tomasol** [comment]
Keep `:<n>` as is, but can we add `:+<n>` and `:-<n>` for relative jumps?
